.sfdevrc.json

```diff
@@ -1,5 +1,5 @@
 {
   "test": {
-    "testsPath": "test/**/*Test.ts"
+    "testsPath": "test/**/*est.ts"
```

Comment:
it's better to just have 1 pattern.  Let's only support *.test.ts.  That is pretty much universal in our plugins.